src/Datasource/QueryInterface.php

```diff
@@ -157,6 +157,7 @@ public function first();
      * Returns the total amount of results for the query.
      *
      * @return int
+     * @deprecated
```

Comment:
I see count() is now covered by the SelectQuery class, so still part of the interface in 5.x?

Reply:
`QueryInterface::count()` isn't deprecated.